mule-gateway/modules/ROOT/pages/mule-gateway-metrics.adoc

```diff
@@ -18,6 +18,28 @@ Even though no sensitive information is collected using this configuration, you
 
 `anypoint.platform.metrics_enabled=false`
 
+[[log-analytics-event-data]]
+== Log Analytics Event Data
+
+Mule Gateway provides `INFO` API analytics event logging to capture data about incoming API requests. 
+
+NOTE: API analytics event logging is only supported in Mule runtime versions 4.4.X released after Feb 1, 2024 and Mule runtime versions 4.6.0 and later. Deleting cached policy assets is not supported in Mule Runtime 4.5.x.
+
+By default, `anypoint.platform.log_analytics_events=false`. Enable event logging by adding the following entry to the `wrapper.conf` file:
+
+[source,conf]
+----
+anypoint.platform.log_analytics_events=true
+----
+
+IMPORTANT: Due to additional log generation, events logging might affect the performance of the request processing.
```

Comment:
Do you want to use this as when we write [NOTE]? Like: 
"
[IMPORTANT]
Due to...
"

Reply:
I wanted to add like a banner, as the asciidocs specification mentions for those admonitions: https://user-images.githubusercontent.com/967328/61221576-d3589e80-a6e6-11e9-975f-54311198caec.png

I used that because I don't know much of asciidoc, maybe it is the same as doing that

Reply:
They actually both do the same thing. So, it's fine.